lib/commons/dom/is-modal-open.js

```diff
@@ -14,6 +16,14 @@ function isModalOpen(options) {
   options = options || {};
   const modalPercent = options.modalPercent || 0.75;
 
+  try {
```

Comment:
What's this for?

Reply:
The creator `detectModal` returns undefined on line `52`, which causes `cache.get` to throw. That looks to be entirely normal behavior with this, so the error is being caught.

I can toss in a comment, check for that specific error type, or both. It doesn't look like we would want to change the return behavior.

Reply:
As above, I misread the date that your comment on undefined was. This try/catch comes out when considering that so I am resolving.